src/libs/actions/Session/index.ts

```diff
@@ -356,6 +357,10 @@ function signOutAndRedirectToSignIn(shouldResetToHome?: boolean, shouldStashSess
     // Wait for signOut (if called), then redirect and update Onyx.
     return signOutPromise
         .then((response) => {
+            if (isSupportal) {
+                // Send event to Fraud Protection backend, otherwise it might consider the user as being suspicious
+                FraudProtection.sendEvent(FRAUD_PROTECTION_EVENT.STOP_SUPPORT_SESSION);
+            }
```

Comment:
When the user performs a logout after supportaling, no API call is made. It just restores the stashed login. So we need to send the event from here instead of from the `FraudMonitoring` middleware.